cli/src/main/resources/hudson/cli/client/Messages_ru.properties

```diff
@@ -0,0 +1,26 @@
+CLI.Usage=Jenkins CLI\n\
+  Применение: java -jar jenkins-cli.jar [-s URL] command [opts...] args...\n\
+  Параметры:\n\
+  \ -s URL              : URL-адрес сервера (по умолчанию используется переменная env JENKINS-URL)\n\
+  \ -webSocket          : Подключится через WebSocket (по умолчанию; хорошо работает с большинством обратных прокси; требуется Jetty)\n\
+  \ -http               : Используется соединение HTTP(S), а не WebSocket\n\
```

Comment:
English version hints me it's the default connection option, maybe it should be mentioned?